src/core/EdFi.DataManagementService.Core/Middleware/RejectResourceIdentifierMiddleware.cs

```diff
@@ -3,22 +3,19 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-using System.Text.Encodings.Web;
 using System.Text.Json;
 using System.Text.Json.Nodes;
 using EdFi.DataManagementService.Core.External.Model;
 using EdFi.DataManagementService.Core.Model;
 using EdFi.DataManagementService.Core.Pipeline;
 using Microsoft.Extensions.Logging;
 using static EdFi.DataManagementService.Core.Response.FailureResponse;
+using static EdFi.DataManagementService.Core.UtilityService;
 
 namespace EdFi.DataManagementService.Core.Middleware
 {
     internal class RejectResourceIdentifierMiddleware(ILogger _logger) : IPipelineStep
     {
-        private static readonly JsonSerializerOptions _serializerOptions =
-            new() { Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping };
-
```

Comment:
Good catch - I'm glad you made this change so that we could stop duplicating the serializer settings everywhere.